requirements/base.in

```diff
@@ -12,3 +12,4 @@ edx-toggles
 event-tracking
 six
 XBlock[django]
+edx-event-routing-backends  # Provides xAPI transforms for aggregator events
```

Comment:
We've done some testing with Aspects + ERB and palm/quince, and I don't think there were any compatibility issues discovered there -- @bmtcril would know more.

But since I had to make [some modifications to ERB](https://github.com/openedx/event-routing-backends/pull/431) to support this change, it won't be compatible with prior ERB versions anyway.